app/components/deduped-hearings-list.js

```diff
@@ -0,0 +1,57 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default class DedupedHearingsListComponent extends Component {
+  // the actions "attribute" to display in the hearings list e.g. "ulurpnumber" or 'name'
+  // this is set when the component is rendered
+  attribute = '';
+
+  hearingsSubmitted = false;
+
+  @computed('hearingsSubmitted', 'project')
+  get dedupedHearings() {
+    const dispositions = this.get('project.dispositions');
+    // hearingsSubmitted is set when the component is rendered
+    const { hearingsSubmitted } = this;
+
+    // an array of objects that have been reduced
+    let deduped;
+
+    // setting a new property on each disposition called hearingActions which is an array of objects
+    // this property hearingActions is initally set to an array of the current disposition's action model
+    // during the reduce, if there is a duplicate in the array of dispositions,
+    // the actions model for that duplicate disposition is pushed into this array
+    dispositions.forEach(function(disposition) {
+      disposition.set('hearingActions', [disposition.action]);
+    });
+
+    // hearingsSubmitted is set when the component is rendered
+    if (hearingsSubmitted) {
+      // reduce method to deduplicate dispositions based on location and date of hearing
+      deduped = dispositions.reduce((accumulatedDisps, currentDisp) => {
+        // object that represents a match between accumulatedDisps and the currentDisp
+        const matchingPropsObject = accumulatedDisps.find(item => item.publichearinglocation === currentDisp.publichearinglocation && item.dateofpublichearing.toString() === currentDisp.dateofpublichearing.toString());
+
+        // if an object exists in accumulatedDisps that matches the currentDisp
+        if (matchingPropsObject) {
+          // iterate through the array of accumulated dispositions
+          accumulatedDisps.forEach(function(disposition) {
+            // check whether an object in accumulatedDisps matches the currentDisp object
+            const matchingObject = disposition.publichearinglocation === currentDisp.publichearinglocation && disposition.dateofpublichearing.toString() === currentDisp.dateofpublichearing.toString();
+
+            if (matchingObject) {
+              // grab the actions model from the currentDisp
+              const currentDispAction = currentDisp.get('action');
+              // push this into the hearingActions array on the matched object in accumulatedDisps
+              disposition.hearingActions.push(currentDispAction);
```

Comment:
awesome. Can we additionally associated each disp with its "duplicate disps"? I.e. add this line:
`disposition.duplicateDisps.push(curredDisp)`

 this will make it easier for the recommendation page to assign  a 'wasquorumpresent' value to each disposition binned under a hearing.

Reply:
will do